src/formats.c

```diff
@@ -54,6 +54,114 @@ static void test_fmt_case(struct fmt_main *format, void *binary,
 	int *plaintext_has_alpha, struct db_salt *dbsalt);
 #endif
 
+static char *fmt_classes[] = { "all", "cpu", "opencl", "dynamic", "mask", "omp", "cpu+omp", NULL };
+
+static char* is_format_class(char *name)
+{
+	int idx = 0;
+
+	if (strchr(name, '*') || strchr(name, '#'))
+		return "wildcards";
```

Comment:
I had forgotten about the `#` wildcard. It's a gem! And lo and behold, it's documented.